src/pages/documentation/adobe-media-analytics/migration-guide.md

```diff
@@ -0,0 +1,114 @@
+import Tabs from './tabs/migration-guide.md'
+
+# Migrating to Adobe Streaming Media for Edge Network Extension
+
+## Configuration Changes
+
+**Media**
+
+| Name | Key | Value | Required |
+| --- | --- | --- | --- |
+| `Collection API Server` | "media.trackingServer" | String | Yes |
+| `Channel` | "media.channel" | String | No |
+| `Player Name` | "media.playerName" | String | No |
+| `Application Version` | "media.appVersion" | String | No |
+
+**Media for Edge Network**
+
+| Name | Key | Value | Required |
+| --- | --- | --- | --- |
+| `Channel` | "edgeMedia.channel" | String | **Yes** |
+| `Player Name` | "edgeMedia.playerName" | String | **Yes** |
+| `Application Version` | "edgeMedia.appVersion" | String | No |
+
+
+## Dependency Changes
+
```

Comment:
nit: There is no underline below this heading. Possible to have some generic text to trigger an underline for the heading?

Same for Install extensions and Import & Register extensions.